weft/handlers.go

```diff
@@ -58,11 +59,11 @@ var compressibleMimes = map[string]bool{
 
 var defaultCsp = map[string]string{
 	"default-src":     "'none'",
-	"img-src":         "'self' *.geonet.org.nz data: https://www.google-analytics.com https://stats.g.doubleclick.net",
+	"img-src":         "'self' *.geonet.org.nz data: https://*.google-analytics.com https://*.googletagmanager.com",
 	"font-src":        "'self' https://fonts.gstatic.com",
 	"style-src":       "'self'",
-	"script-src":      "'self'",
-	"connect-src":     "'self' https://*.geonet.org.nz https://www.google-analytics.com https://stats.g.doubleclick.net",
+	"script-src":      "'self' https://*.googletagmanager.com",
```

Comment:
Maybe possible to use [this script](https://github.com/GeoNet/www-geonet/blob/main/cmd/www-geonet/assets/assets/js/analytics.js#L18) to insert google js, without add to `script-src`

Reply:
Have removed change to `script-src`, still works